ci/qemu/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -ex
+disk=${1:-vda.qcow2}
+qemu-system-arm -M virt -m 1024 \
```

Comment:
Why you need to start `qemu-system-arm` in `install.sh` as well? https://github.com/apache/incubator-mxnet/pull/12094/files#diff-c33e17b2cfe6dd45dc7c0ef96e01d521R24

Reply:
Install is to prepare the base image.